src/core/public/chrome/chrome_service.tsx

```diff
@@ -262,6 +264,7 @@ export class ChromeService {
           isLocked$={getIsNavDrawerLocked$}
           branding={injectedMetadata.getBranding()}
           survey={injectedMetadata.getSurvey()}
+          workspaces={workspaces}
```

Comment:
@raintygao you may check the latest commit from @SuZhou-Joe , it should be `workspaces.client.workspaceList$` here which is an observable.

Reply:
@SuZhou-Joe Do we have an observable for getting features of current workspace?

Reply:
Currently we don't have an observable way for getting features of current workspace

Reply:
For now there is not, but a good reminder here, maybe I can add one by piping workspaceList$ and currentWorkspaceId$.